api/python/cellxgene_census/tests/experimental/pp/test_stats.py

```diff
@@ -0,0 +1,184 @@
+from typing import Any, Union
+
+import numpy as np
+import pandas as pd
+import pytest
+import tiledbsoma as soma
+from scipy import sparse
+
+import cellxgene_census
+from cellxgene_census.experimental import pp
+
+
+@pytest.mark.experimental
+@pytest.mark.live_corpus
+@pytest.mark.parametrize("axis", [0, 1])
+@pytest.mark.parametrize(
```

Comment:
Missing edge case that needs testing:   query that returns nothing

Reply:
There is a dedicated test for that: `test_mean_variance_empty_query`

Reply:
so sorry - I missed that!